gst/nnstreamer/elements/gsttensor_debug.c

```diff
@@ -399,6 +399,8 @@ _gst_tensor_debug_output (GstTensorDebug * self, GstBuffer * buffer)
   UNUSED (self);
   UNUSED (buffer);
   /** @todo NYI: do the debug task */
+
+  ml_loge ("new tokens");
```

Comment:
Looks like this with tensor_debug
```sh
$ gst-launch-1.0 videotestsrc num-buffers=1 ! video/x-raw,format=RGB,width=10,height=10,framerate=0/1 ! videoconvert ! video/x-
raw, format=RGB ! tensor_converter ! tensor_generator ! tensor_debug ! filesink location=tmp.bin
Setting pipeline to PAUSED ...
Pipeline is PREROLLING ...

** (gst-launch-1.0:306273): CRITICAL **: 10:41:23.073: new tokens
Pipeline is PREROLLED ...
Setting pipeline to PLAYING ...
New clock: GstSystemClock

** (gst-launch-1.0:306273): CRITICAL **: 10:41:23.073: new tokens

** (gst-launch-1.0:306273): CRITICAL **: 10:41:23.073: new tokens

** (gst-launch-1.0:306273): CRITICAL **: 10:41:23.073: new tokens

** (gst-launch-1.0:306273): CRITICAL **: 10:41:23.073: new tokens

** (gst-launch-1.0:306273): CRITICAL **: 10:41:23.073: new tokens
Got EOS from element "pipeline0".
Execution ended after 0:00:00.000154442
Setting pipeline to NULL ...
Freeing pipeline ...
```

Reply:
```
videotestsrc num-buffers=1 ! video/x-raw,format=RGB,width=10,height=10,framerate=0/1 ! videoconvert ! video/x-
raw, format=RGB 
```
==>
```
videotestsrc num-buffers=1 ! videoconvert ! videoscale ! videorate ! video/x-raw,format=RGB,width=10,height=10,framerate=0/1
```
is better and robust to format changes.
(if the first capsfilter declares caps that videotestsrc does not support, your pipeline will fail.)

Reply:
@niley7464 ,
This pipeline would be useful to get user input.
```
gst-launch-1.0 fdsrc fd=0 ! application/octet-stream ! tensor_converter ! other/tensors,format=flexible ! tensor_debug ! fakesink
```

tensor_debug:
```
static GstFlowReturn
gst_tensor_debug_transform_ip (GstBaseTransform * trans, GstBuffer * buffer)
{
  GstMemory *in_mem;
  GstMapInfo in_info;
  GstTensorMetaInfo meta;
  GstTensorDebug *self = GST_TENSOR_DEBUG (trans);
  gsize hsize = 0;

  _gst_tensor_debug_output (self, buffer);
  in_mem = gst_buffer_peek_memory (buffer, 0);
  if (!gst_memory_map (in_mem, &in_info, GST_MAP_READ)) {
    return GST_FLOW_ERROR;
  }

  gst_tensor_meta_info_parse_header (&meta, in_info.data);
  hsize = gst_tensor_meta_info_get_header_size (&meta);

  nns_logd ("Received data: %s", (gchar *) (in_info.data + hsize));

  return GST_FLOW_OK;
}
```